File: core-java-practice/gcr-codebase/programming-elements/CanvertKiloToMiles.java
import java.util.*;

public class CanvertKiloToMiles {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        //take input from user
        double kilo = sc.nextDouble();
        double miles = kilo * 0.621371;//convert kilo to miles
        System.out.print(miles);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/CanvertKiloToMiles.cs


class CanvertKiloToMiles {
    static void Main(String[] args) {
        //take input from user
        double kilo = Convert.ToDouble(Console.ReadLine());
        double miles = kilo * 0.621371;//convert kilo to miles
        Console.WriteLine(miles);
    }
}
